### PR TITLE
Add 2 collection concatenation

### PR DESCRIPTION
**Pull request by ccedmendoza**

### CODE REVIEW

**Comment by drosofff** [comment]
That looks nice. Could you improve the documentation. The examples are ok, but try to make their use more consistent and work on the terminology that is a bit "jargon"

**Comment by ccedmendoza** [comment]
what do you mean by more consistent?

File: tools/concat_multi_datasets/catWrapper.xml [hunk 1]
```diff
@@ -331,13 +331,23 @@
...
 -----
 
 **What it does**
 
 Concatenates datasets and paired collections with multiple options:
 
- - It's possible select either a concatenation by strand, by pair or a whole collection concatenation, when the input is a paired collection.
+ - When the input is a paired collection:
+
+   - concatenation by strand : all the forward files are concatenated then all the reverse files are concatenated to finish with a single forward - reverse pair
```
Comment (drosofff) [inline tools/concat_multi_datasets/catWrapper.xml]: forward and reverse datasets are concatenated separately and a list with a single forward - reverse dataset pair is returned

File: tools/concat_multi_datasets/catWrapper.xml [hunk 2]
```diff
@@ -331,13 +331,23 @@
...
+
+   - concatenation by pair : all the pairs are concatenated to finish with a collection containing single files
```
Comment (drosofff) [inline tools/concat_multi_datasets/catWrapper.xml]: forward - reverse dataset pairs are concatenated and a simple dataset collection is returned

File: tools/concat_multi_datasets/catWrapper.xml [hunk 3]
```diff
@@ -331,13 +331,23 @@
...
+
+   - whole collection concatenation : every file is concatenated to finish with a single file
```
Comment (drosofff) [inline tools/concat_multi_datasets/catWrapper.xml]: all datasets in the collection are concatenated and a single dataset is returned

File: tools/concat_multi_datasets/catWrapper.xml [hunk 4]
```diff
@@ -331,13 +331,23 @@
...
+
+ - When the inputs are 2 collections: files are concatenated 2 by 2
```
Comment (drosofff) [inline tools/concat_multi_datasets/catWrapper.xml]: When the inputs are 2 collections:  datasets are concatenated in pairwise combination (?)

**Comment by drosofff** [comment]
ok 👍

**Pull request merged**